app/controllers/exam_authorization_requests_controller.rb

```diff
@@ -0,0 +1,31 @@
+class ExamAuthorizationRequestsController < ApplicationController
+  def show
+    @authorization_request = ExamAuthorizationRequest.find(params[:id])
+    current_user.read_notification! @authorization_request
+  end
+
+  def create
+    authorization_request = ExamAuthorizationRequest.create! authorization_request_params
+    current_user.read_notification! authorization_request.exam_registration
+    send_updated_notice!
+    redirect_to root_path
+  end
+
+  def update
+    ExamAuthorizationRequest.update params[:id], authorization_request_params
+    send_updated_notice!
+    redirect_to root_path
+  end
+
+  private
+
+  def send_updated_notice!
```

Comment:
Just nitpicking here, but this method name is a bit misleading. We could just have to two different notices, one for request created and another for updated and avoid this method altogether